<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/images/settingBackground.png);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="playerNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>玩家名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxMemLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>最大内存</string>
   </property>
  </widget>
  <widget class="QLabel" name="javaPathLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>Java路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerNameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="autoSetRadioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自动设置</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="ManualSetRadioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxMemLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoRearchJavaPathPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>自动搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="javaPathLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowsePushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAndExitPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="aboutSoftLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>86</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;mclauncheraboutsoft&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;关于软件&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
